src/Microsoft.DotNet.Build.Tasks/PackageFiles/tests.targets

```diff
@@ -159,13 +160,63 @@
           />
   </Target>
 
+  <PropertyGroup Condition="'$(BuildingUAPVertical)' == 'true'">
+    <CommonPriFile Condition="'$(CommonPriFile)' == ''">$(RuntimePath)/resw/resources.pri</CommonPriFile>
+    <TestSpecificPriFile>$(TestPath)\resources.pri</TestSpecificPriFile>
+  </PropertyGroup>
+
+
+
+  <Target Name="CalculateResWFiles" Condition="'$(BuildingUAPVertical)' == 'true'">
+    <ItemGroup>
+      <TestResWFiles Include="$(TestResourcesFolderPath)\*.resw" />
+      <CommonResWFiles Include="$(ResourcesFolderPath)\*.resw" />
+    </ItemGroup>
+  </Target>
+
+  <Target Name="MakeCommonResourcesPriFile"
+          DependsOnTargets="CalculateResWFiles"
+          Condition="'$(BuildingUAPVertical)' == 'true'"
+          Inputs="@(CommonResWFiles)"
+          Outputs="$(CommonPriFile)">
+
+    <MakeResourcesPriFile ResWFiles="@(CommonResWFiles)"
+                          RunnersPriFilePath="$(TestHostRootPath.Replace('/', '\'))\Runner\resources.pri"
```

Comment:
I did the normalization to avoid errors within makepri because I didn't know if that could cause any exceptions, but I can keep them with no normalization if it doesn't hurt makepri and you rather have them without normalization.